### PR TITLE
[DOCFIX] Update StressBench.md

### PR DESCRIPTION
**Pull request by apc999**
### What changes are proposed in this pull request?

Update stressbench doc

### Why are the changes needed?

mark it experimental

### Does this PR introduce any user facing changes?

na

### CODE REVIEW

File: docs/en/operation/StressBench.md [hunk 1]
```diff
@@ -24,90 +24,91 @@ the Alluxio system. The following benchmark tools are currently supported:
...
   Omit to use all available job workers.
-* `--cluster-start-delay`: specify the start delay before starting the benchmark, used to
-  synchronize the jobs.
-* `--java-opt`: additional JVM options
-* `--profile-agent`: profile agent to use during benchmark
-* `--bench-timeout`: the maximum time a benchmark should run for
+* `--cluster-start-delay`: specify the start delay before starting the benchmark, used to synchronize jobs.
+* `--java-opt`: specify additional JVM options
 * `--help`: print usage, description and benchmark-specific options of a benchmark
+* `--profile-agent`: profile agent to use during benchmark
```
Comment (HelloHorizon) [inline docs/en/operation/StressBench.md:71]: can you remove this profile agent option?

**Review by HelloHorizon — CHANGES_REQUESTED** [review]
@apc999 thanks for the improvement! Left one minor comment.

File: docs/en/operation/StressBench.md (hunk 1 above)
Comment (apc999) [reply docs/en/operation/StressBench.md:71]:
```suggestion
```

**Review by HelloHorizon — APPROVED** [review]
LGTM

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**